homeassistant/components/aemet/const.py

```diff
@@ -206,7 +210,7 @@
         name="Condition",
     ),
     SensorEntityDescription(
-        key=ATTR_FORECAST_PRECIPITATION,
+        key=ATTR_FORECAST_NATIVE_PRECIPITATION,
```

Comment:
We shouldn't change keys or entity unique_id. These keys should be independent of the weather entity api. The keys should be integration specific.

Reply:
You're right, the change of the unique_id is not motivated.